sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.0.0-beta.6 (Unreleased)
 
+### Breaking Changes
+
+Removed Share Lease related API due to it not supported in recent service versions.
```

Comment:
Probably we don't need to add the reason why we introduced the breaking change. It might be better to list the removed APIs (because the end user might not know all of them and would try to search for a missing API and found it in the changelog).

Reply:
The reason is for the customer/maintainer to understand why this change is made, more of a justification than an explanation.
I think listing the removed APIs is a valid suggestion. I'll change this.